x-pack/solutions/search/plugins/enterprise_search/public/applications/applications/components/search_applications/search_application_indices_flyout.tsx

```diff
@@ -85,7 +55,10 @@ export const SearchApplicationIndicesFlyout: React.FC = () => {
         }
       ),
       render: (indexName: string) => (
-        <EuiLink {...SearchIndicesLinkProps(indexName)}>{indexName}</EuiLink>
+        <SearchApplicationViewIndexLink
+          indexName={indexName}
+          dataTelemetryId={'entSearchApplications-list-viewIndex'}
```

Comment:
the `{}` are not needed if you're passing a static string.
```suggestion
          dataTelemetryId='entSearchApplications-list-viewIndex'
```

Reply:
updated in 359b77ee2b258d5a6b7751e9cb6748f490be0200